fairytale.md

```diff
@@ -15,4 +20,4 @@ The average Squonk is 7 meters tall and screams loud.
 >  What does the **world** he lives in **look like**?
 >  What happened between it and the *Drowsloks*?
 
-`Well... your turn now.`
```

Comment:
Why is this line removed? It is present in my master. Something went wrong with the merge. Could you add back?